### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tsmoyo**
added a some details in README.md

### CODE REVIEW

File: README.md
Comment (phoexer) [inline README.md]:
```suggestion
[![Gmail](mailto: thabomoyo47@gmail.com)]
```

File: README.md
Comment (phoexer) [inline README.md]:
```suggestion
### Hi there 👋, I'm Thabo and I'm learning to build websites.
```

**Review by phoexer — CHANGES_REQUESTED** [review]
This is ok, I left two suggestions.
So the protocol is usually when some one leaves suggestions you make changes to the branch then update the PR and re request a review. Normally we don't merge into main until you have approval, so let's try that

**Comment by tsmoyo** [comment]
how is this?

**Review by phoexer — APPROVED** [review]
lgtm

**Pull request merged**
